conda/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "eva-sub-cli" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  path: ../
```

Comment:
I'll change this before we merge, but meanwhile this should allow you to test locally, e.g.
```
conda activate <env>
cd eva-sub-cli/conda  # directory containing the recipe
conda build .
conda install --use-local eva-sub-cli
```

Reply:
Discussed this a bit, for now we'll leave it as is to enable local building.
Note that to get the above commands to work, you also need to either have the correct channels in your config (running `conda config --add channels bioconda conda-forge`), or specify the channels explicitly in both `build` and `install` commands (adding `-c bioconda -c conda-forge` to both).
The command in the readme right now should work fine once it's actually on conda.